### PR TITLE
1cloudfile.com

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/138148

### CODE REVIEW

File: BaseFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -3070,7 +3070,7 @@ mixfaucet.com#%#//scriptlet("abort-on-property-read", "adblockDetector")
 @@||cloudfront.net/?*=$domain=1cloudfile.com
 !#endif
 ! For DNS filtering
-||1cloudfile.com$replace=/Request\("https:\/\/d192r5l88wrng7\.cloudfront\.net"/Request\("https:\/\/example\.com"/
+||1cloudfile.com$replace=/("https:\/\/d192r5l88wrng7\.cloudfront\.net"/("https:\/\/example\.com"/
```
Comment (Alex-302) [inline BaseFilter/sections/antiadblock.txt:3073]: Suggested another solution in PM, because this does not help.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: BaseFilter/sections/antiadblock.txt (hunk 1 above)
Comment (Alex-302) [reply BaseFilter/sections/antiadblock.txt:3073]: @jellizaveta 

image

Comment (Sergey-Lyapin) [reply BaseFilter/sections/antiadblock.txt:3073]: @jellizaveta

**Comment by jellizaveta** [comment]
Cannot reproduce, even with DNS enabled

**Pull request closed without merging**